lib/Parser/DocumentParser.php

```diff
@@ -505,7 +505,7 @@ private function parseLine(string $line): bool
                 }
 
                 if ($this->lineChecker->isFieldOption($line)) {
-                    if ($this->fieldOption !== null) {
+                    if ($this->directive !== null && $this->fieldOption !== null) {
```

Comment:
So we silently don't set the option? Should maybe an error end up in the error manager or something?

Reply:
Non-existent directives already throw warnings. I don't see a reason, why a directive that has field options should throw a stronger errror then a directive without. Non-existent directive is non-existent directive in my eyes?

Reply:
I wasn't aware warnings were already thrown. In that case, this should be fine 👍